server/src/test/java/org/elasticsearch/bootstrap/BootstrapChecksTests.java

```diff
@@ -146,7 +164,9 @@ public void testExceptionAggregation() {
                     containsString("bootstrap check failure [1] of [2]:"),
                     containsString("first"),
                     containsString("bootstrap check failure [2] of [2]:"),
-                    containsString("second")
+                    containsString("second"),
+                    containsString("For more information see " +
+                        "[https://www.elastic.co/guide/en/elasticsearch/reference/")
```

Comment:
nit: combine string constants (may need reformatting)

```suggestion
                    containsString("For more information see [https://www.elastic.co/guide/en/elasticsearch/reference/")
```